style.css

```diff
@@ -463,23 +412,288 @@ header {
 }
 
 .submit-contact {
-  width: 131px;
-  margin-left: 98px;
-  margin-top: -12px;
+  width: 40.06%;
 }
 
 footer {
-  padding: 25px 95px;
+  width: 23em;
+  padding: 25px 0 5px 0;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
   border-top: 1px solid #dfe1e6;
   margin: 124px 0 8px 0;
+  box-sizing: border-box;
 }
 
 .footerIcon {
-  filter: invert(80%);
+  filter: invert(70%);
 }
 
-.headerLine {
+.footerLine {
   border-top: 5px solid #0d151e;
-  margin-left: 36px;
   width: 134px;
 }
+
+@media (min-width: 768px) {
```

Comment:
- [ ] as the previous reviewer asked, try to fix the gap in the 680px wide screen the page should change smoothly without gaps, I suggest doing that 

![image](https://user-images.githubusercontent.com/61790587/165821855-66ec946d-dfa0-4d47-b455-6417ade957e3.png)